docs: using bake with remote and local metadata definition (#395)
docker/metadata-action: README.md

          targets: build
```

> If you're building a [remote Bake definition](https://docs.docker.com/build/bake/remote-definition/)
> using a [Git context](https://github.com/docker/bake-action?tab=readme-ov-file#git-context),
> you can specify the metadata bake definition with the `cwd://` prefix:

Reviewer: WDYT about this

```suggestion
If you're building a [remote Bake definition](https://docs.docker.com/build/bake/remote-definition/)
using a [Git context](https://github.com/docker/bake-action?tab=readme-ov-file#git-context),
you must specify the location of the metadata-only bake file using a `cwd://` prefix:
```

Author: Looks good, thanks @dvdksn
